src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsGroupComponent.scala

```diff
@@ -73,6 +73,20 @@ trait RawlsGroupComponent {
       loadGroup(findGroupByName(groupRef.groupName.value))
     }
 
+    def subGroupEmailsQuery: Query[(Rep[String], Rep[String]), (String, String), Seq] = {
```

Comment:
Make this return type `GroupMemberEmailQuery` (or a better name) and add tests for these in RawlsGroupComponentSpec